packages/fetchai/contracts/oracle_client/contract.py

```diff
@@ -48,6 +48,7 @@ def get_query_transaction(
         contract_address: Address,
         from_address: Address,
         query_function: str,
+        amount: int = 0,
```

Comment:
nm 0 is fine

Reply:
Yeah, good catch, I just set this to zero so that the transaction would fail if the skill is not configured with the correct amount, but it might make more sense to make this a required argument to the function.